src/Build.UnitTests/BackEnd/IntrinsicTask_Tests.cs

```diff
@@ -314,6 +314,26 @@ public void ItemKeepDuplicatesFalse()
             Assert.Single(group);
         }
 
+        [Fact]
+        public void ItemKeepDuplicatesFalseTwoDuplicatesAtOnce()
+        {
+            string content = ObjectModelHelpers.CleanupFileContents(@"
+            <Project ToolsVersion='msbuilddefaulttoolsversion' xmlns='msbuildnamespace'>
```

Comment:
There's no need for ToolsVersion and xmlns, let's remove these and keep the tests minimal